engine/runtime/src/main/scala/org/enso/interpreter/instrument/DebuggerMessageHandler.scala

```diff
@@ -91,7 +91,15 @@ class DebuggerMessageHandler extends MessageEndpoint {
                   Debugger.createEvaluationFailure(exceptionWithTraces)
                 )
               case Right(value) =>
-                sendToClient(Debugger.createEvaluationSuccess(value))
+                node.showObject(value) match {
+                  case Left(error) =>
+                    Logger[this.type].error(
+                      s"Failed to call `to_text` on computation result (${error.getMessage}), falling back to Java representation."
+                    )
```

Comment:
I imagine this will be shown to the user. Do they need to know these details?

Reply:
This is shown only in unexpected cases - for example if `to_text` panics. I think it makes sense to log this. Maybe could change the level to warning though. Do you think that's better?

Reply:
(This is not for cases like Java objects, but for errors when running `to_text`, fallback for Java should be silent)

Reply:
Ok, makes sense